src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -691,6 +691,24 @@ public override ImmutableArray<MethodSymbol> ExplicitInterfaceImplementations
             }
         }
 
+        public override Accessibility DeclaredAccessibility
+        {
+            get
+            {
+                if ((object)_explicitInterfaceType != null && ContainingType.IsInterface)
+                {
+                    LazyMethodChecks();
+                    if (!_lazyExplicitInterfaceImplementations.IsDefaultOrEmpty)
+                    {
+                        Debug.Assert(_lazyExplicitInterfaceImplementations.Length == 1);
+                        return _lazyExplicitInterfaceImplementations[0].DeclaredAccessibility;
```

Comment:
>return _lazyExplicitInterfaceImplementations[0].DeclaredAccessibility; [](start = 24, length = 70)

I am going to add the PROTOTYPE comment here and in other similar places in one of the next PRs.